_manual/translate.md

```diff
@@ -7,13 +7,13 @@ order: 50
 
 <h3>Help Flyspray to get localized to your native language</h3>
 
-Want your language strings updated with the next Flyspray release? Pick your language file on https://github.com/Flyspray/flyspray/tree/master/lang , edit and make a pull request.
+Want your language strings updated with the next Flyspray release? Pick your language file on [the FlySpray github repository](https://github.com/Flyspray/flyspray/tree/master/lang), edit and make a pull request.
```

Comment:
Writing Flyspray with the big S in the middle isn't used anymore. There are still some old translation files with that, but most updated languages and the 1.0 logo and website do not use that.

Reply:
ok